src/services/notifications/index.ts

```diff
@@ -191,13 +194,58 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }
   }
 
+  const sendBalance = async (account: Account): Promise<void> => {
```

Comment:
In the other `sendNotification` methods in this file we pass in the pre-processed values to be included in the notification, whereas in this new method we're determining the values internally using `Wallet` and `User` concepts.

I'm wondering if this breaks one of the boundaries we've setup and whether we should be determining these outside this function and passing them in as arguments? (or is this something we already plan to do in follow-up PRs?)

Reply:
good point, I can at least compute the balance outside of this function